cmd/flakechecker/flake_config.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
+	"io"
+	"os"
+	"regexp"
+)
+
+// flakeDetectionPolicyConfig represents configuration used by flakechecker to evaluate failed tests.
+//
+// It contains the following fields:
+// match_job_name - name of the job that should be evaluated by flakechecker. i.e. (branch should be evaluated, but main not)
+// ratio_job_name - job name that should be used for ratio calculation. i.e. we take main branch test runs as base for evaluation of flake ratio
+// test_name_regex - regex used to match test names. Some test names contain detailed information (i.e. version 4.4.4), but we want to use ratio for all tests in that group (i.e. 4.4.z). Using regex allow us to group tests differently.
+// classname - class name of the test that should be isolated. With this option we can isolate single flake test from suite and isolate only that one from the rest.
```

Comment:
I don't understand this description, maybe an example would help? 🤔

Reply:
I agree. I also don't understand it. 😅